Anytype/Sources/PresentationLayer/Modules/ObjectType/ObjectTypeView.swift

```diff
@@ -9,13 +9,52 @@ struct ObjectTypeView: View {
     
     var body: some View {
         content
-            .task { await model.setup() }
+            .animation(.default, value: model.templates)
+            .task { await model.setupSubscriptions() }
     }
     
     private var content: some View {
         VStack {
-            AnytypeText(model.title, style: .title)
-            AnytypeText("TBD", style: .subheading).foregroundColor(.Text.secondary)
+            Rectangle().frame(height: 48).foregroundStyle(Color.Text.secondary) // Navbar placeholder
+            VStack(spacing: 0) {
+                Spacer.fixedHeight(32)
+                header
+                templates
+                Spacer()
+            }.padding(.horizontal, 20)
```

Comment:
I think templates don't need to have horizontal padding - they should scroll after the screen edges)